### DISCUSSION TITLE
No motion events ring doorbell

### DISCUSSION THREAD

**Discussion by cornevandepol** [tsightler/ring-mqtt#825] [Support Q&A] [answered]
Somehow I stopped receiving motion events for my ring doorbell for a while now (I think after an update but I am not sure). I started digging and I got as far as that the ring-mqtt does not get doorbell motion events while it does get motion events from another camera. Could be that the problem lies deeper that ring-mqtt but I don't know how to proceed from here. Maybe there is just something "wrong" with my ring settings ?! (I tried to find similar questions but could not find the right one)

I started the rinq-mqtt container in debugging mode and see that the doorbell is "discovered" intially also I see these message frequently:
![image](https://github.com/tsightler/ring-mqtt/assets/115914609/89687213-867d-4fb1-a2c7-cc67381f5294)
I also get an updated snapshot in home assistant once in a while.

Even though I am "dancing" in front of the camera there are no motion events but I do get them in the ring app:
![image](https://github.com/tsightler/ring-mqtt/assets/115914609/63803c3c-1224-4139-ba2e-3df0fb343e75)

Motion detection is enabled (Sorry, everything is in dutch...)
![image](https://github.com/tsightler/ring-mqtt/assets/115914609/e4ad44e2-22c6-4c61-93fb-dd6d9ca7b040)

Maybe good to mention I do have motion "zones" (areas? don't know how this is translated) defined as well as smart notifications to only send a notification if a person is detected (but I can't remember I changed something here recently).

I can imagine more information is required and/or I should search somewhere different but thanks anyway. Besides this minor glitch I really appreciate this project!

**Comment by tsightler** [comment, marked as answer]
If the push notifications are not arriving there's not much troubleshooting that can be done since ring-mqtt can't act on something it's not receiving.

Overall, troubleshooting steps are to stop the addon, go to Ring website and make sure to delete all ring-mqtt clients from Ring Control Center, start the addon and re-authenticate.  Re-authenticating this way will re-register the Firebase push-receiver with Ring and typically fixes any issues with push notifications not going through.

**Reply by cornevandepol** [reply]
I hadn't thought of that, but it worked! Thanks a lot.
